src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,33 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH = 10;
+    private K[] keys = (K[]) new Object[MAX_LENGTH];
+    private V[] values = (V[]) new Object[MAX_LENGTH];
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < MAX_LENGTH; i++) {
+            if (keys[i] == key) {
```

Comment:
```suggestion
            if (keys[i] == key) {
```
What if it's not the same object but they are equal?